backend/routes/Config.types.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable node/no-unpublished-import */
+import type app from '../../conf/app.json';
+import type root from '../../conf/config.json';
+import type logging from '../../conf/logging.json';
+import type dbOptions from '../../conf/db.options.json';
+import type userConfig from '../../conf/userConfig.json';
+import type manager from '../../conf/manager.json';
+import type { Mode } from '../serverSettings';
+
+export type UserConfig = typeof userConfig;
+
+export interface Config {
+    app: typeof app & typeof root & typeof logging & { db: { options: typeof dbOptions } } & typeof userConfig;
```

Comment:
WDYT about extracting the `app` type to make the interface look more readable?

```suggestion
type App = typeof app & typeof root & typeof logging & { db: { options: typeof dbOptions } } & typeof userConfig;

export interface Config {
    app: App;
```

It's a minor change but it can always bring us some value in the future 🎉

Reply:
Yes, makes sense, adjusted it and reused in `ClientConfig` interface.